.github/workflows/pr_check_load_test.yml

```diff
@@ -13,18 +13,22 @@ on:
       - main
 
 env:
-  LOAD_TEST_RESOURCE: "rbac-test"
+  LOAD_TEST_RESOURCE: "mohit-1"
```

Comment:
Can we use some other name? or get it from repo variables?

Reply:
yes, please update the secrets or env in the repo

Reply:
done